sidebars.js

```diff
@@ -271,6 +271,15 @@ const sidebars = {
             'en/integrations/data-visualization/zingdata-and-clickhouse',
           ],
         },
+        {
+          type: 'category',
+          label: 'No-Code platforms',
```

Comment:
Not sure we need it in the sidebar. We definitely need it on https://github.com/ClickHouse/clickhouse-docs/blob/main/docs/en/integrations/index.mdx. Could you add, please?